src/augraphy/augmentations/dirtyrollers.py

```diff
@@ -66,6 +66,8 @@ def __call__(self, data, force=False):
                 image = cv2.rotate(image, cv2.ROTATE_90_CLOCKWISE)
 
             image = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
+            
+            # where is this transform?
```

Comment:
@tbrettallen so we should remove the entire self.transform in dirtyrollers augmentation? Right now i can see it is still in the code, and couldn't locate the definition of self.transform.

Reply:
This `transform` method call is left over from before [this commit](https://github.com/sparkfish/augraphy/commit/ea0970932b49288b9893e728d8c4442693b825e4) when it was removed. Probably is why the default pipeline has `DirtyRollersAugmentation(p=0.0)` set in the `post_phase`; it doesn't work anymore. I'll make an issue for this.